doc/user/content/release-notes.md

```diff
@@ -149,6 +149,9 @@ changes that have not yet been documented.
 
 - Add `SELECT` statement support to [`TAIL`](/sql/tail).
 
+- **Breaking change.** Change the output of [`format_type`](/sql/functions/#system-information-func)
```

Comment:
nit: breaking changes go at the top of the release notes for whichever release they're in